clients/core/src/publicPages/application/ApplicationLoginPage.tsx

```diff
@@ -37,7 +38,9 @@ export const ApplicationLoginPage = (): JSX.Element => {
     mutationFn: (application: PostApplication) => {
       return postNewApplicationExtern(phaseId ?? 'undefined', application)
     },
-    onSuccess: () => {
+    onSuccess: (data) => {
+      // Assume the response contains a boolean "confirmationMailSent"
+      setConfirmationMailSent(data.confirmationMailSent)
```

Comment:
should it default to `false` if the `confirmationMailSent` is not in the response?

```suggestion
      setConfirmationMailSent(data.confirmationMailSent ?? false)
```

Reply:
A boolean value automatically defaults to false if it is not present.